### ISSUE TITLE
Scrolling glitch with linked panes

### ISSUE THREAD

**Issue by hiimsergey** [chetachiezikeuzor/Yin-and-Yang-Theme#59]
Hello, I am really grateful for the theme, it is my favorite now, loving it so far. My issue is a glitch that occurs when scrolling on linked panes.
![glitchvideo](https://user-images.githubusercontent.com/91432388/134820699-704a720f-b302-4650-9f35-0c1f0fbe8ead.gif)

This problem was [already found by another user](https://github.com/SlRvb/Obsidian--ITS-Theme/issues/5) with the ITS theme and Obsidianite but the exactly the same thing happens with this one. 
They also found that it happens only when...
- you scroll the **editor** pane while the linked one is in **preview**
- the note contains bullet points, it starts glitching when the points reach the top

Later the developer updates ITS which removed the issue permanently but I don't know what code they added and if I did then I wouldn't know how to utilize it.

**Comment by chetachiezikeuzor** [comment]
Yes, I have received an [issue very similar to this one](https://github.com/chetachiezikeuzor/Yin-and-Yang-Theme/issues/53). Unfortunately, the relationship lines are the culprit. I'm still working on an alternative version that retains the same styling so I will notify you once I have this done. For now, it may be best to toggle the relationship lines off. 

https://user-images.githubusercontent.com/79069364/133859593-5cfdf49c-da8c-471c-8b50-b3fc24ea4eec.mp4

**Comment by hiimsergey** [comment]
Thank you very much! :) 
Take all the time you need.
